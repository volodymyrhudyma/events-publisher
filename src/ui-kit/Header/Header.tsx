import React, { ReactNode } from 'react';
import { Wrapper } from './styled/Header.styled';

interface IHeaderProps {
	children: ReactNode;
}

const Header = ({ children }: IHeaderProps) => <Wrapper>{children}</Wrapper>;

export default Header;
