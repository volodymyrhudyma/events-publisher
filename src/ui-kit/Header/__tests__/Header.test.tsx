import React from 'react';
import { shallow } from 'enzyme';
import Header from '../Header';

describe('Header component', () => {
  it('should render correctly', () => {
    const component = shallow(<Header>Header</Header>);
    expect(component).toMatchSnapshot();
  });
});
