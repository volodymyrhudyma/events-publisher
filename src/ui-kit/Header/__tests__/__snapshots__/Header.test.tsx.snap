// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render correctly 1`] = `
<styled.header>
  Header
</styled.header>
`;
